CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 ## v0.23.0
 - Disabled WalletConnect v1
 
+## v0.22.1 
+- Changed Support Service from Telegram to Discord
```

Comment:
Why do we change the v0.22.1 release notes in the v0.23.0 release?

Reply:
I created this branch because I forgot to merge the 0.22.1 hotfix into dev. These are the changes from the previous merge.